graphql-spring-boot-autoconfigure/src/main/java/graphql/kickstart/autoconfigure/web/servlet/GraphQLWebAutoConfiguration.java

```diff
@@ -301,15 +302,13 @@ public GraphQLConfiguration graphQLServletConfiguration(
       @Autowired(required = false) GraphQLResponseCacheManager responseCacheManager,
       @Autowired(required = false) AsyncTaskDecorator asyncTaskDecorator,
       @Autowired(required = false) @Qualifier("graphqlAsyncTaskExecutor") Executor asyncExecutor) {
-    long asyncTimeout =
-        graphQLServletProperties.getAsyncTimeout() != null
-            ? graphQLServletProperties.getAsyncTimeout()
-            : asyncServletProperties.getTimeout();
+    long asyncTimeout = Optional.ofNullable(graphQLServletProperties.getAsyncTimeout()) //
+        .orElse(asyncServletProperties.getTimeout()).toMillis();
```

Comment:
There may be a situation when both are set to `null`, and this would result in NPE.  (The configuration properties could be made `@Validated`, and Duration fields marked as  `@NotNull`)

Reply:
Thank you. It's dealt with